### ISSUE TITLE
"back-end"? Any better name suggestsions?

### ISSUE THREAD

**Issue by willbarton** [cfpb/back-end#4]
This is the repo where we, back-end developers at CFPB, can discuss and write the standards and best practices for the work that we do. Our organization creates that logical separation between front-end and back-end development, so that's why the name "back-end" was chosen for this repo.

But it doesn't have to be that way. Does anyone have a suggestion that would also capture the wide variety of work that falls under our purview?

I'd be partial to "software development", but that seems to imply developers that don't share our roll don't develop software, which I don't think is fair.

"back-end" is also consistent with the existing [front-end repository](https://github.com/cfpb/front-end).

**Comment by rosskarchner** [comment]
At one point, I advocated for replacing FEWD/BEWD with "application" and "api"-- but I think that's just how to world was looking in 2013-ish.

Maybe "server-side"?

**Comment by ascott1** [comment]
Ideally we can combine everything from the front-end repo and this one into a single resource, since folks often touch parts of the stack outside of their specialty.

**Comment by willbarton** [comment]
Yes, I think ideally this all gets folded into one space. I do think it's good to have some separate space to hash out some basics first though. I don't think the naming is terribly important, to be honest.
